howso/utilities/utilities.py

```diff
@@ -81,7 +77,7 @@ def trainee_from_df(df, features: Optional[Mapping[str, Mapping]] = None,
 
     Returns
     -------
-    howso.openapi.models.Trainee
+    Dict
```

Comment:
The return of this method should also be a dict. And the import from howso.engine should be removed.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60